doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md

```diff
@@ -235,6 +235,7 @@ A configuration example could be:
     cygnus-ngsi.sinks.mysql-sink.mysql_port = 3306
     cygnus-ngsi.sinks.mysql-sink.mysql_username = myuser
     cygnus-ngsi.sinks.mysql-sink.mysql_password = mypassword
+    cygnus-ngsi.sinks.mysql-sink.mysql_maxPoolSize = 5
```

Comment:
A parte de incluirlo en la lista de parametros para el sink habria que añadirlo a la tabla con su descripcion:
https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md#configuration

Asi como a todo lo relacionado con docker:
Configuración por defecto: https://github.com/telefonicaid/fiware-cygnus/blob/master/docker/cygnus-ngsi/agent.conf#L46
Entry point (lectura de ENV vars): https://github.com/telefonicaid/fiware-cygnus/blob/master/docker/cygnus-ngsi/cygnus-entrypoint.sh#L68

Reply:
Added parameter description to the table in f014b0b
 https://github.com/telefonicaid/fiware-cygnus/blob/1598_MySQL-Implement-Connection-Pooling/docker/cygnus-ngsi/agent.conf#L213